templates/workload.template.yaml

```diff
@@ -142,12 +142,12 @@ Parameters:
     Type: String
   AdminServerSubdomain:
     Default: ''
-    Description: 'Subdomain for the admin server. Fully qualified domain name for the admin server will be created by adding the DomainName parameter value as a suffix. For example, if the provided value is ''adminserver'' and the DomainName parameter value is ''corp.com'', then the fully qualified domain name for the admin server will be ''adminserver.corp.com''.'
+    Description: 'Sub-domain for the admin server. Fully qualified domain name for the admin server will be created by adding the DomainName parameter value as a suffix. For example, if the provided value is ''adminserver'' and the DomainName parameter value is ''corp.com'', then the fully qualified domain name for the admin server will be ''adminserver.corp.com''.'
     Type: String
   AuthToken:
-    AllowedPattern: '[a-zA-Z0-9]*'
-    ConstraintDescription: 'Must be 16-128 characters, including at least three uppercase, lowercase, nonalphanumeric, and digits.'
-    Default: DuoRedisAuthToken
+    AllowedPattern: '^(?=(.*[^A-Za-z0-9]){3,})(?=(.*[A-Z]){3,})(?=(.*\d){3,})(?=(.*[a-z]){3,}).{16,128}$'
+    ConstraintDescription: 'Must be 16-128 characters, including at least three of uppercase, lowercase, nonalphanumeric, and digits.'
+    Default: DuoRedisAuthToken123!@#
```

Comment:
The default AuthToken needs to be changed, the '@' symbol isn't allowed: 

```
AuthToken
Reserved parameter. The password used to access a password protected server.

Password constraints:

Must be only printable ASCII characters.

Must be at least 16 characters and no more than 128 characters in length.

Nonalphanumeric characters are restricted to (!, &, #, $, ^, <, >, -, ).
```

Also, the redis-cluster.template.yaml child template needs to be updated with this: 

```
      CacheParameterGroupName: 'default.redis5.0'
```

Reply:
Fixed